common/ladder/index.ts

```diff
@@ -1,17 +1,18 @@
+import { RaceStats } from '../races'
 import { SbUser, SbUserId } from '../users/sb-user'
 
 /**
  * A ranked player for a particular matchmaking ladder. Contains information about their play
  * history and current rating.
  */
-export interface LadderPlayer {
+export type LadderPlayer = {
```

Comment:
Would generally always prefer using an `interface` and `extends`, type unions should be saved for usage in most cases (e.g. "I need this function to accept the combination of these 2 types") not for normal type declaration